tika-pipes/tika-grpc/src/main/proto/tika.proto

```diff
@@ -0,0 +1,92 @@
+// Copyright 2015 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+package tika;
+
+option java_multiple_files = true;
+option java_package = "org.apache.tika";
+option java_outer_classname = "TikaProto";
+option objc_class_prefix = "HLW";
+
+
+service Tika   {
+  rpc CreateFetcher(CreateFetcherRequest) returns (CreateFetcherReply) {}
```

Comment:
Could we document these RPCs to understand high level behaviour? For example, if I try to create a fetcher which already exists, what is the expected reply? Is it an error response on the RPC, will CreateFetchReply have error identifying information?

If CreateFetcher was made idempotent, could we collapse these into a single RPC (UpdateFetcher), which either creates, or updates, or noops (no changes despite call) to the Fetcher?

Don't want to over complicate the Tika side of course, but curious if we can improve the client interface.

Reply:
done.